diff --git a/visa/iga/IGALibrary/api/iga.cpp b/visa/iga/IGALibrary/api/iga.cpp
index d52eba4..7582ed6 100644
--- a/visa/iga/IGALibrary/api/iga.cpp
+++ b/visa/iga/IGALibrary/api/iga.cpp
@@ -617,7 +617,7 @@ public:
 const char *iga_version_string()
 {
 #define IGA_EXTERNAL_INTERNAL "ext"
-    return IGA_VERSION_STRING "-" IGA_EXTERNAL_INTERNAL " (" __DATE__ ")";
+    return IGA_VERSION_STRING "-" IGA_EXTERNAL_INTERNAL;
 }
 
 
