diff --git a/IGC/AdaptorOCL/dllInterfaceCompute.cpp b/IGC/AdaptorOCL/dllInterfaceCompute.cpp
index 2908dfe..aab5d2c 100644
--- a/IGC/AdaptorOCL/dllInterfaceCompute.cpp
+++ b/IGC/AdaptorOCL/dllInterfaceCompute.cpp
@@ -66,7 +66,7 @@ SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 #include "common/LLVMWarningsPush.hpp"
 #include "AdaptorOCL/SPIRV/SPIRVconsum.h"
 #include "common/LLVMWarningsPop.hpp"
-#include "AdaptorOCL/SPIRV/SPIRV-Tools/include/spirv-tools/libspirv.h"
+#include <spirv-tools/libspirv.h>
 #include "AdaptorOCL/SPIRV/libSPIRV/SPIRVModule.h"
 #include "AdaptorOCL/SPIRV/libSPIRV/SPIRVValue.h"
 #endif
